superset/views/base.py

```diff
@@ -320,7 +320,8 @@ class CsvResponse(Response):
     """
     Override Response to take into account csv encoding from config.py
     """
-    charset = conf.get('CSV_EXPORT').get('encoding', 'utf-8')
+    _charset = conf.get('CSV_EXPORT').get('encoding', 'utf-8')
+    charset = 'utf-8' if _charset.lower() == 'utf-8-sig' else _charset
```

Comment:
Out of interest why are you using the `utf-8-sig` encoding rather than `utf-8`?

Reply:
utf-8-sig is utf-8 encoding with BOM, and this is microsoft Excel detect utf-8 encoding. 
In this scenarios, if use Excel open to east asian charset by Excel,  excel show garbled.